Changelog.md

```diff
@@ -1,34 +1,42 @@
-## 4.0.0 - UNRELEASED
+## 4.0.0 - 2024-03-13
 
-## Changed
+### Added
+
+-   Reintroduced triggers functionality.
```

Comment:
Will the user know what the triggers functionality is?

Reply:
yes